src/adapters/cf-worker/adapter.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  canonicalizeHeaders,
+  flatHeaders,
+  NormalizedRequest,
+  NormalizedResponse,
+  addHeader,
+} from '../../runtime/http';
+
+export async function workerConvertRequest(
+  req: Request,
+): Promise<NormalizedRequest> {
+  const body = await new Response(req.body).text();
```

Comment:
I _think_ the proper processing would be to look at the `Content-Type` header and see if it has a `charset` directive, and use that to use appropriate processing, probably by calling `Request.arrayBuffer()` and then converting that accordingly.

Granted, it _might_ be overkill, since these are API requests?  What's the likelihood that they will not be `UTF-8`?